components/post_markdown/post_markdown.jsx

```diff
@@ -59,7 +59,9 @@ export default class PostMarkdown extends React.PureComponent {
 
         this.props.pluginHooks.forEach((o) => {
             if (o && o.hook) {
-                message = o.hook(post, message);
+                if (post) {
```

Comment:
0/5 and not a big deal but why not:

```javascript
if (o && o.hook && post) {
```
instead of nesting another if statement